src/dockerflow/fastapi/__init__.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import APIRouter
+from fastapi.routing import APIRoute
+
+from .views import heartbeat, lbheartbeat, version
+
+router = APIRouter(
+    tags=["Dockerflow"],
+    routes=[
+        APIRoute("/__lbheartbeat__", endpoint=lbheartbeat, methods=["GET", "HEAD"]),
+        APIRoute("/__heartbeat__", endpoint=heartbeat, methods=["GET", "HEAD"]),
+        APIRoute("/__version__", endpoint=version, methods=["GET"]),
+    ],
+)
+"""This router adds the Dockerflow views."""
```

Comment:
Move this to the top of the file?

Reply:
This is the way Sphinx `autodoc` documents module attributes (docstring below)